tests/parser/data/failures.csv

```diff
@@ -0,0 +1,18 @@
+"SELECT 'a%' LIKE 'a!%' ESCAPE '!', 'a%' LIKE 'a!' || '%' ESCAPE '!'"
```

Comment:
What's the failure in this one?

Reply:
@adamziel I think this one is due to [PIPES_AS_CONCAT](https://dev.mysql.com/doc/refman/8.4/en/sql-mode.html#sqlmode_pipes_as_concat) SQL mode and the query extraction script not tracking & saving SQL modes. The other ones, I think some need small grammar adjustments and some are due to bad query extraction. I think it won't be a big problem to get to 0 queries, I only wanted to wrap up the PR last week, but I'll get to this.